activestorage/README.md

```diff
@@ -24,7 +24,7 @@ class User < ApplicationRecord
 end
 
 # Attach an avatar to the user.
-user.avatar.attach(io: File.open("~/face.jpg"), filename: "avatar.jpg", content_type: "image/jpg")
+user.avatar.attach(io: File.open(File.expand_path("~/face.jpg")), filename: "avatar.jpg", content_type: "image/jpg")
```

Comment:
Think `File.expand_path` on top of `File.open` makes the example too complex. Let's go with `File.open("/path/to/face.jpg")` same for the racecar examples.

Reply:
@kaspth Thanks for the comment. I've fixed it :)